medusa/tv/series.py

```diff
@@ -548,6 +549,13 @@ def release_required_words(self):
         """Return release ignore words."""
         return [v for v in (self.rls_require_words or '').split(',') if v]
 
+    @staticmethod
+    def normalize_status(series_status):
+        """Return a normalized status given current indexer status."""
+        if not series_status:
+            return 'Unknown'
```

Comment:
But if series_status is None I can't make it lower() as i do in the next line

So what to do?